sklearn/utils/tests/test_validation.py

```diff
@@ -135,9 +135,11 @@ def test_check_array():
     X = [[1, 2], [3, 4]]
     X_csr = sp.csr_matrix(X)
     assert_raises(TypeError, check_array, X_csr)
-    # ensure_2d
+    # ensure_2d (False case)
```

Comment:
not really crucial but `ensure_2d=False` and `ensure_2d=True` reads better to me.